langchain/src/vectorstores/index.ts

```diff
@@ -8,3 +8,4 @@ export { PineconeStore } from "./pinecone.js";
 export { VectorStore, SaveableVectorStore } from "./base.js";
 export { SupabaseVectorStore } from "./supabase.js";
 export { PrismaVectorStore } from "./prisma.js";
+export { TypeORMVectorStore } from "./typeorm.js";
```

Comment:
Can't export from here as it requires an optional dep - will break people currently (and mistakenly) importing from here.

Reply:
ok